gradle/conf/checkstyle.xml

```diff
@@ -188,6 +188,7 @@
         <module name="ParameterNumber">
             <property name="max" value="5"/>
         </module>
+        <module name="FinalParameters"/>
```

Comment:
Enforce parameters for methods, constructors, and catch blocks to be final.
http://checkstyle.sourceforge.net/config_misc.html#FinalParameters